bundles/best_practices.rst

```diff
@@ -116,23 +116,20 @@ files are going to be part of the repository.
 The following classes and files have specific emplacements (some are mandatory
 and others are just conventions followed by most developers):
 
-===============================  =============================  ================
-Type                             Directory                      Mandatory?
-===============================  =============================  ================
-Commands                         ``Command/``                   Yes
-Controllers                      ``Controller/``                No
-Service Container Extensions     ``DependencyInjection/``       Yes
-Event Listeners                  ``EventListener/``             No
-Model classes [1]                ``Model/``                     No
-Configuration                    ``Resources/config/``          No
-Web Resources (CSS, JS, images)  ``Resources/public/``          Yes
-Translation files                ``Resources/translations/``    Yes
-Templates                        ``Resources/views/``           Yes
-Unit and Functional Tests        ``Tests/``                     No
-===============================  =============================  ================
-
-[1] See :doc:`/doctrine/mapping_model_classes` for how to handle the
-mapping with a compiler pass.
+===============================  ========================================  ==========
+Type                             Directory                                 Mandatory?
+===============================  ========================================  ==========
+Commands                         ``Command/``                              Yes
+Controllers                      ``Controller/``                           No
+Service Container Extensions     ``DependencyInjection/``                  Yes
+Doctrine entities and documents  ``Entity/`` (ORM) or ``Document/`` (ODM)  No 
+Event Listeners                  ``EventListener/``                        No
+Configuration                    ``Resources/config/``                     No
+Web Resources (CSS, JS, images)  ``Resources/public/``                     Yes
+Translation files                ``Resources/translations/``               Yes
```

Comment:
I'm not sure because those directories are optional when using annotations. In this case, I'd also like to have a dedicated article to explain these things, as stated in a previous comment.